scripts/requirements-extras.txt

```diff
@@ -19,7 +19,9 @@ Pillow
 imgtool>=1.7.1
 
 # used by nanopb module to generate sources from .proto files
-protobuf
+# Google introduced breaking changes in protobuf v4.21.0, use v3 for now
+# see https://developers.google.com/protocol-buffers/docs/news/2022-05-06
+protobuf<4
```

Comment:
Why not just say `<v4.2x.0` for whatever value of `x` is correct?

Reply:
I thought `<4` was cleaner, as only the major part is relevant here.